cfme/tests/intelligence/reports/test_reports_schedules.py

```diff
@@ -45,8 +46,46 @@ def test_schedule_queue(schedule, appliance):
         casecomponent: Reporting
         caseimportance: high
         initialEstimate: 1/10h
+        tags: report
     """
 
     schedule.queue()
     view = schedule.create_view(ScheduleDetailsView)
-    view.flash.assert_message('The selected Schedule has been queued to run')
+    view.flash.assert_message("The selected Schedule has been queued to run")
+
+
+def test_reports_disable_enable_schedule(schedule_data, appliance):
+    """
+    Polarion:
+        assignee: pvala
+        casecomponent: Reporting
+        caseimportance: high
+        initialEstimate: 1/10h
+        tags: report
+    """
+    schedules = appliance.collections.schedules
+    schedule = schedules.create(**schedule_data)
+    schedules.disable_schedules(schedule)
+    assert not schedule.enabled
+    schedules.enable_schedules(schedule)
+    assert schedule.enabled
+    schedule.delete()
+
```

Comment:
@valaparthvi Teardown: ```schedule.delete()``` should be before ```assert``` statement.
Otherwise if assertion fails then it will not perform ```schedule.delete()```.

Reply:
Good catch!